platform/ext/target/nordic_nrf/nrf5340/partition/flash_layout.h

```diff
@@ -51,8 +51,8 @@
  */
 
 /* Size of a Secure and of a Non-secure image */
-#define FLASH_S_PARTITION_SIZE                (0x30000)       /* S partition: 192 kB*/
-#define FLASH_NS_PARTITION_SIZE               (0x40000)       /* NS partition: 256 kB*/
+#define FLASH_S_PARTITION_SIZE                (0x41000)       /* S partition: 260 kB*/
+#define FLASH_NS_PARTITION_SIZE               (0x30000)       /* NS partition: 192 kB*/
```

Comment:
I am not sure I like this @oyvindronningstad 
I want the flash layout to be compatible with zephyr default upstream partitioning, so NSPE needs to start on 0x40000

Reply:
Well, like it or not we need the extra space for secure_fw in configurations like `ConfigRegression.cmake`

    Linking C executable unit_test/tfm_s.axf
    Memory region         Used Size  Region Size  %age Used
               FLASH:      252896 B       258 KB     95.72%
                 RAM:       81352 B       256 KB     31.03%

Reply:
OK, so we need a clean solution here
- first, any changes here need to be reflected in the comment section above, which gives the overview of the flash layout. Lines 21-44.
- then, I am wondering whether 129kB for NSPE are sufficient, e.g. for NCS applications (I know that most of upstream  Zephyr samples for nRF5340 will fit in 192kB). If 129kB is too small, I wonder if we could use some #ifdef's here and have "conditional" flash layout
-

Reply:
It looks clean now.